src/MarginTrading.Common/Services/MtSlackNotificationsSender.cs

```diff
@@ -32,6 +32,11 @@ public MtSlackNotificationsSender(ISlackNotificationsSender sender, string appNa
             await _sender.SendAsync(ChannelTypes.MarginTrading, sender, GetSlackMsg(message));
```

Comment:
ChannelTypes.MtMmRisks is not used anymore. 
And if you added SendRawAsync may be it is better to change SendAsync and use ChannelTypes.Monitor with SendRawAsync?